platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/snapshot/DefaultStateSnapshotManager.java

```diff
@@ -117,7 +124,9 @@ public SignedStateFileManager(
      *                            this method call and this method will release the reservation when it is done
      * @return the result of the state saving operation, or null if the state was not saved
      */
-    public @Nullable StateSavingResult saveStateTask(@NonNull final ReservedSignedState reservedSignedState) {
+    @Override
```

Comment:
The methods in the class are now `@Override`s, and duplicate the javadocs that exist in the `StateSnapshotManager` interface

Reply:
deduped documentation